### PR TITLE
 Update French Translation

### PR DESCRIPTION
**Pull request by gandf**
Update French Translation

### CODE REVIEW

**Comment by sdottaka** [comment]
Thank you!

File: Translations/WinMerge/French.po [hunk 1]
```diff
@@ -42,22 +43,22 @@ msgstr "65001"
...
 #, c-format
 msgid "Copy &from Left"
-msgstr ""
+msgstr "Copier depuis la Gauche"
 
 #: Merge.rc:8AF7E68
 #, c-format
 msgid "Copy fro&m Right"
-msgstr ""
+msgstr "Coper depuis la Droite"
```
Comment (vlakoff) [inline Translations/WinMerge/French.po:61]: Typo: should be "Copier"

File: Translations/WinMerge/French.po [hunk 2]
```diff
@@ -368,7 +374,7 @@ msgstr "Décompression &automatique"
 #: Merge.rc:714B45FD
 #, c-format
 msgid "&Edit with Unpacker..."
-msgstr ""
+msgstr "&Éditer avec le décompresseur..."
```
Comment (vlakoff) [inline Translations/WinMerge/French.po:377]: Anglicisme: should be "Modifier" (and there is probably the same mistake at other places)

File: Translations/WinMerge/French.po [hunk 3]
```diff
@@ -4313,23 +4484,33 @@ msgstr "Erreur lors de la comparaison des fichiers."
...
 msgstr ""
 "Ces fichiers utilisent différents type de fin de ligne.\n"
 "\n"
-"Voulez-vous considérer l'égalité des caractères de fin de ligne pour cette comparaison ?\n"
+"Voulez-vous considérer l'égalité des caractères de fin de ligne pour cette "
+"comparaison ?\n"
 "\n"
-"Notez bien: Si vous voulez toujours considérer une telle égalité. Définnissez l'option: Ignorer les différences de fin de ligne à la fenêtre des Préférences sous l'onglet Comparaison (accessible avec le menu Edition/Préférences)."
+"Notez bien: Si vous voulez toujours considérer une telle égalité. "
+"Définnissez l'option: Ignorer les différences de fin de ligne à la fenêtre "
```
Comment (vlakoff) [inline Translations/WinMerge/French.po:4511]: Typo: should be "Définissez"

**Comment by sdottaka** [comment]
@vlakoff Could you make a pull request?

**Pull request merged**